collated/test/A0127014W.md

```diff
@@ -0,0 +1,258 @@
+# A0127014W
+###### \java\guitests\CollapseExpandCommandTest.java
+``` java
+package guitests;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class CollapseExpandCommandTest extends AddressBookGuiTest{
+
+    @Test
+    public void collapse_alreadyCollapsed() {
```

Comment:
the test name should indicate what is expected for this test case. recollect test names should have 3 parts